src/app/locationForm/location-name/LocationName.jsx

```diff
@@ -48,16 +57,20 @@ class LocationName extends Component {
 }
 
 const mapStateToProps = (state, ownProps) => {
-  const { locationId } = ownProps.location.state;
+  const locationId = ownProps.match.params.locationId;
```

Comment:
I'm surprised eslint doesn't complain about this. On the back-end, at least, the airbnb rules enforce the slightly-more-ES7-idiomatic `const { locationId } = ownProps.match.params;`.

Reply:
eslint passes, and doesn't complain. I'm not sure why.